docs/process_updating_mce_hypershift.md

```diff
@@ -0,0 +1,35 @@
+# Updating Hypershift
+
+The ACM is packaged as an OCM Addon and deploy to the service cluster with fleet manager.
+
+ACM includes the multicluster engine operator (MCE) which is the component that really packages the hypershift operator and it's related images. ACM provides the policy functionality, and MCE provides all the cluster lifecycle functionality.
```

Comment:
ACM includes the multicluster engine (MCE) which is the operator that provides the hypershift operator, the cluster lifecycle functionalities. ACM additionally provides the policy and governance functionalities.